src/sql/pg/create.sql

```diff
@@ -0,0 +1,147 @@
+--
+-- PostgreSQL
+--
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = "warning";
+SET search_path = "public", "pg_catalog";
+--
+-- Sequences
+--
+
+CREATE SEQUENCE "N_CASES_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+CREATE SEQUENCE "N_NOTIFICATION_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+--
+-- Tables
+--
+CREATE TABLE "N_LU_CASE_STATUS"
+(
+    "CASE_STATUS_ID"             INTEGER NOT NULL ,
+    "CASE_STATUS_CODE"           CHARACTER VARYING(20) NOT NULL ,
+    "CASE_STATUS_DESC"           CHARACTER VARYING(255) NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  CURRENT_USER  NOT NULL 
+);
+
+CREATE INDEX "XPKN_LU_CASE_STATE" ON "N_LU_CASE_STATUS"
```

Comment:
Based on the email I sent out from the discussion with Abraham, the N is supposed to be separated by an underscore and before the other prefix.